BlueprintUICommonControls/Sources/AccessibilityElement.swift

```diff
@@ -37,6 +40,7 @@ public struct AccessibilityElement: Element {
         accessibilityFrameCornerStyle: CornerStyle = .square,
         customActions: [AccessibilityElement.CustomAction] = [],
         customContent: [AccessibilityElement.CustomContent] = [],
+        userInputLabels: [String]? = nil,
```

Comment:
The underlying property is declared as implicitly unwrapped:
```swift
var accessibilityUserInputLabels: [String]! { get set }
```
And the docs say the default value is an empty array.
Should `userInputLabels` be non-optional, and the default be `[]`?
It's probably fine - just want to double-check.

Reply:
Ah yes, of course, love an implicitly unwrapped property.

Reply:
It's so weird that this one is implicitly unwrapped, but others like `customActions` are optional. 🤷🏼